azure-pipelines.yml

```diff
@@ -22,6 +22,9 @@ steps:
     echo "##vso[task.setvariable variable=docker_path;]$docker_path"
   displayName: 'Set version number'
 
+- script: docker pull $(docker_path):latest || true
+  displayName: "Pull latest docker image to cache"
```

Comment:
nit-picking, but this is "from cache", right? ;)

Reply:
I think we pull it to the cache don't we as there is nothing in there until we populate it? So probably from Docker Hub to cache, and then from cache during the build? Hence cache_from in the az.yml?

Reply:
Ah, ok. Ya, that makes sense!